header-validator/validate-os.js

```diff
@@ -76,10 +91,8 @@ function validate(str, paramChecks) {
   return { errors, warnings }
 }
 
-export function validateRegisterOsSource(str) {
-  return validate(str, {})
-}
-
-export function validateRegisterOsTrigger(str) {
-  return validate(str, {})
+export function validateOsRegistration(str) {
```

Comment:
With this change we also have to update https://github.com/WICG/attribution-reporting-api/blob/1779655910cbc11fd0fde8d6cc850245425cb1e9/validate-headers.html#L92.

Reply:
Thanks. I still keep both `os-source` and `os-trigger`, but please let me know if you prefer to have only one.